docs/Developer-Help.md

```diff
@@ -1,23 +0,0 @@
-# Developer help
-
-This is a wiki for helping developers working in this application. It has detailed explanations of the application and known gotchas. 
-
-## Code execution overview
-
-Webpack creates two bundles, the first being `server` that has an entrypoint of `/src/index.js` and the second being `client` which has an entry point of `/src/client.js`
-
-The entry points to the application in terms of code execution are as follows: 
-- `npm run build:ci` executes the webpack configs
-- `webpack.config.js` is executed and then executes both `webpack.config.client.js` and `webpack.config.server.js`
-- `webpack.config.client.js` then executes `src/client.js` 
-- `webpack.config.server.js` then executes `src/index.js` 
-- `src/index.js` then executes `src/server/index.jsx`
-
-## `.env` is showing in my `git status`
```

Comment:
This issue is often seen in new contributors' initial PRs. So to improve the onboarding experience, I'd suggest keeping this paragraph within an FAQ section of a Getting-Started.md or Contributing.md.

Reply:
Copied into contributing.md